docs/commands.md

```diff
@@ -4,7 +4,7 @@
 
 ### TS.CREATE
 
-Create a new time-series.
+Create a new time-series. TS.CREATE must be called to initialize database before you can add any values with TS.ADD.
```

Comment:
Two small comments:
1. you don't have to call TS.CREATE, TS.ADD will create it automatically if the key doesn't exists.
2. In Redis we call a set of keys database, while a key can hold multiple data items (or samples in RTS case). its a bit confusing when comparing to other databases.